pkgs/development/python-modules/filedepot/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, anyascii
+, buildPythonPackage
+, fetchFromGitHub
+, flaky
+, mock
+, paste
+, pillow
+, pymongo
+, pytestCheckHook
+, pythonOlder
+, requests
+, sqlalchemy
+}:
+
+buildPythonPackage rec {
+  pname = "filedepot";
+  version = "0.9.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "amol-";
+    repo = "depot";
+    rev = "refs/tags/${version}";
+    hash = "sha256-OJc4Qwar3sKhKKF1WldwaueRG7FCboWT2wXYldHJbPU=";
+  };
+
+  propagatedBuildInputs = [
+    anyascii
+  ];
+
+  nativeCheckInputs = [
+    flaky
+    mock
+    paste
```

Comment:
I think `paste` is not needed if tests in `examples` are not disabled. (I don't think it's a big deal though if we keep it.)

Reply:
OK, let's keep it for now. If needed then it can always be removed in the future.

Thanks for checking. Much appreciated.